src/storage/fileset.proto

```diff
@@ -21,6 +21,14 @@ message DeleteFile {
   string path = 1;
 }
 
+// TODO: Src and dst naming?
+// TODO: Append?
+message CopyFile {
+  string fileset_id = 1;
+  string src = 2;
```

Comment:
Is this a single file or a prefix?  We could also do path ranges.

Reply:
It has the same semantics as a PFS path (i.e. directory if it ends in a trailing slash and a file otherwise). We created a ticket for revisiting these semantics for the fileset API that we should consider doing before shipping this. I think it would be ideal for this to be a file filter. I'll put a note about this.